style.css

```
@@ -0,0 +1,300 @@
+    flex-direction: row;
+}
+
+.flex_column {
+    display: flex;
+    flex-direction: column;
+}
+
+/* ADD A TASK FORM */
+
+#new_task_form {
+    margin-top: 5rem;
+    margin-bottom: 5rem;
+    margin-left: 21rem;
+    gap: 0.5rem;
+}
+
+
+#new_task_text {
+    font-weight: 600;
+    width: 30rem;
+    padding: 1rem;
+    border: none;
+    border-radius: 1rem;
+    color: white;
+    background-color: #FFAEA5;
+}
+
+#select_progress {
+    font-weight: 600;
+    padding: 1rem;
+    border: none;
+    border-radius: 1rem;
+    color: white;
+    background-color: #FFDBCC;
+}
+
+#new_task_btn {
+    font-size: 2rem;
+    font-weight: 600;
+    width: 4rem;
+    border: none;
+    background-color: white;
+    border-radius: 1rem;
+    cursor: pointer;
+}
+
+
+/* LIST CONTAINER */
+
+.list_container {
+    gap: 1rem;
+}
+
+.list_column {
+    width: 100%;
+    height: 100%;
+    margin: 2rem;
+}
+
+.tasks_container{
+    min-height: 30vh;
+}
+
+.title {
+    height: 4rem;
+    align-items: center;
+    justify-content: space-between;
+    padding-inline: 2rem 1rem;
+    border-radius: 1rem;
+    background-color: #FFC8A2;    
+}
+
+.title h2 {
+    font-size: 2rem;
+    font-weight: 600;
+    color: white;
+}
+
+.sort_btn {
+    height: 3rem;
+    width: 4rem;
+    margin-right: 0.5rem;
+    background-image: url(./resources/sort.png);
+    background-repeat: no-repeat;
+    background-size: 20px 20px;
+    background-position: center;
+    background-color: white;
+    border: none;
+    border-radius: 1rem;
+    cursor: pointer;
+}
+
+.counter {
+    height: 3rem;
+    width: 4rem;
+    font-size: 1.3rem;
+    font-weight: 700;
+    justify-content: center;
+    align-items: center;
+    border: none;
+    border-radius: 1rem;
+    background-color: white;
+}
+
+
+
+
+/* NEW CARD STYLE */
+
+.card_task {
+    position: relative;
+    height: auto;
+    padding: 1rem;
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    border: 0.4px solid #FF968A;
+    border-radius: 1rem;
+    background-color: white;
+    cursor: move;
+}
+
+.card_task h2 {
+    margin-bottom: 0.5rem;
+}
+
+.card_task p {
+    font-size: 1.1rem;
+}
+
+.card-content {
+    position: absolute;
+    display: none;
+    margin-top: 1.2rem;
+}
+
+.card-content p {
+    font-weight: 700;
+    color: #55CBCD;
+}
+
+.card_task:hover .card-content {
+    display: block;
+}
+
+
+/* EDIT BUTTONS */
+
+.card_edit {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    
+}
+
+.edit_btn, .delete_btn {
+    height: 3rem;
+    width: 4rem;
+    border: none;
+    border-radius: 1rem;
+    background-color: white;
+    background-repeat: no-repeat;
+    background-position: center;
+    cursor: pointer;
+}
+
+.edit_btn {
+    background-image: url(./resources/edit_note.svg);
+    background-size: 33px 33px;
+}
+
+.delete_btn {
+    background-image: url(./resources/delete.png);
+    background-size: 25px 25px;
+}
+
+
+
+.hidden {
+    display: none;
+}
```

some other name would be more accurate I think to distinguish `visibility: hidden` and `display: none` properties, they work different (btw, check it)
```suggestion
.xxx {
    display: none;
}
```